Microsoft.Toolkit.Uwp.UI.Controls.Input/TokenizingTextBox/TokenizingTextBox.xaml

```diff
@@ -95,10 +111,7 @@
                                 Margin="{StaticResource TokenizingTextBoxPresenterMargin}"
                                 Footer="{TemplateBinding Footer}"
```

Comment:
Would we want `Footer` be also abstracted here? Otherwise it'll be in the TextBox part again, right?

Reply:
Funnily, in my testing, setting the footer does not actually change anything. So should we move the footer out of the ItemsControl like the header too?

Reply:
Yeah, I think while we're at it here, it makes sense. :)

Reply:
Done!